docs/developers/index.md

```diff
@@ -1,4 +1,4 @@
-# Contributing resources
+# Contributing
 
 Here you can find resources about the contributing workflow, governance of the
 project and the onboarding of new contributors.
```

Comment:
```suggestion
Here you can find resources about the contributing workflow for both code to napari core and documentation of the project, as well as information for onboarding of new core developers.
```
I think this better reflects the current status quo? see:
https://output.circle-artifacts.com/output/job/bb445cdd-3861-417d-95da-ff83eb566578/artifacts/0/docs/_build/developers/index.html

Looks awesome tho!

Reply:
Sounds good -thanks!